src/main/java/core/basesyntax/PasswordValidator.java

```diff
@@ -1,7 +1,18 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.PasswordValidationException;
+
 public class PasswordValidator {
-    public boolean validate(String password, String repeatPassword) {
-        return false;
+    private static final int MIN_PASSWORD_LENGTH = 10;
+
+    public boolean validate(String password, String repeatPassword)
+            throws PasswordValidationException {
+        if (password != null && repeatPassword != null
+                && (password.equals(repeatPassword)
```

Comment:
let's think if we need `repeatPassword != null`
`password.equals(repeatPassword)` already covers this case